src/current/v23.2/molt-fetch.md

```diff
@@ -0,0 +1,413 @@
+| `--continuation-token`                        | Restart fetch at a specific table, using the specified continuation token, if the process encounters an error. `--fetch-id` must be specified. For details, see [Fetch continuation](#fetch-continuation).                                                                                                                                                                                                                |
+| `--direct-copy`                               | Enables [direct copy mode](#fetch-mode), which copies data directly from source to target without using an intermediate store.                                                                                                                                                                                                                                                                                            |
+| `--export-concurrency`                        | Number of shards to use for data export. **Note:** This number will multiply the number of tables being moved by `--table-concurrency`. Ensure your machine has sufficient resources to handle thi level of concurrency.<br><br>**Default:** `4`                                                                                                                                                                          |
+| `--fetch-id`                                  | Restart fetch process corresponding to the specified ID. If `--continuation-file-name` or `--continuation-token` are not specified, fetch restarts for all failed tables.                                                                                                                                                                                                                                                 |
+| `--flush-rows`                                | Number of rows before the source data is flushed to intermediate files.                                                                                                                                                                                                                                                                                                                                                   |
+| `--flush-size`                                | Size (in bytes) before the source data is flushed to intermediate files.                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Not sure how we can call this out here, but if the user specifies both, we pick the one that comes first.